tests/utils.go

```diff
@@ -356,6 +356,49 @@ func getPodsByLabel(label, labelType, namespace string) (*k8sv1.PodList, error)
 	return pods, nil
 }
 
+func GetProcessName(pod *k8sv1.Pod, pid string) (output string, err error) {
+	virtClient, err := kubecli.GetKubevirtClient()
+	if err != nil {
+		return
+	}
+
+	fPath := "/proc/" + pid + "/comm"
+	output, err = ExecuteCommandOnPod(
```

Comment:
@matosatti 
I think this command doesn't exist anymore and moved to `kubevirt.io/kubevirt/tests/exec`
 undefined: ExecuteCommandOnPod

Probably something that got mixed up during rebase
Can you please take a look?